samples/CommunityToolkit.Maui.Sample/Pages/Base/BaseGalleryPage.cs

```diff
@@ -40,12 +40,10 @@ async void HandleSelectionChanged(object? sender, SelectionChangedEventArgs e)
 		var collectionView = (CollectionView)sender;
 		collectionView.SelectedItem = null;
 
-		if (e.CurrentSelection.FirstOrDefault() is not SectionModel sectionModel)
+		if (e.CurrentSelection.FirstOrDefault() is SectionModel sectionModel)
 		{
-			return;
+			await Shell.Current.GoToAsync(AppShell.GetPageRoute(sectionModel.ViewModelType));
```

Comment:
If I'm not wrong it's safe to add the ConfigureAwait here
```suggestion
			await Shell.Current.GoToAsync(AppShell.GetPageRoute(sectionModel.ViewModelType)).ConfigureAwait(false);
```

Reply:
You're not wrong; adding `.ConfigureAwait(false)` here would add a performance improvement. 

As a personal rule-of-thumb when implementing an MVVM architecture, I try to avoid using `.ConfigureAwait(false)` in the View layer since the View code typically _does_ require the UI Thread. 

Since `.ConfigureAwait(false)` does not return to the UI thread, any code added afterwards in a future PR would not be on the main thread. 

In the Services + ViewModel layers, however, I _always_ use `.ConfigureAwait(false)` 💯 

Long story short, I try to avoid `.ConfigureAwait(false)` in the View layer to avoid future bugs where future code may need to be running on the UI Thread.

Reply:
Got it, yeah I suggested it because this is the last async call in the method, so doesn't matter which thread will run. But I understand your point, in terms that this might lead developers - who use this example as a reference - to dragons